scripts/templates/helper.py

```diff
@@ -695,6 +695,14 @@ def make_param_lines(namespace, tags, obj, py=False, decl=False, meta=None, form
         name = _get_param_name(namespace, tags, item)
         if py:
             tname = get_ctype_name(namespace, tags, item)
+            # Handle fptr_typedef
+            if "fptr_typedef" in meta:
+                fptr_types = list(meta['fptr_typedef'].keys())
+                fptr_names = list()
+                for fptr_type in fptr_types:
+                    fptr_names.append(subt(namespace, tags, fptr_type))
```

Comment:
Do we really need to build a list here? Can't we just do the substitution then check if the result is `tname` and if so reassign `tname` to `c_void_p` then execute the loop e.g.

```
for fptr_type in fptr_types:
  if tname == subt(namespace, tags, fptr_type):
    tname == "c_void_p"
    break
```